R/stat-contour.R

```diff
@@ -142,7 +142,7 @@ StatContourFilled <- ggproto("StatContourFilled", Stat,
 
     isobands <- withr::with_options(list(OutDec = "."), xyz_to_isobands(data, breaks))
     names(isobands) <- pretty_isoband_levels(names(isobands))
-    path_df <- iso_to_polygon(isobands, data$group[1])
+    path_df <- iso_to_geom(isobands, data$group[1], "polygon")
```

Comment:
```suggestion
    path_df <- iso_to_geom(isobands, data$group[1], geom = "polygon")
```

Just for symmetry with the above change